app.js

```diff
@@ -20,3 +21,162 @@ hamburger.addEventListener('click', trigger);
 navItems.forEach((trig) => {
   trig.addEventListener('click', trigger);
 });
+
+const portfolioSection = document.getElementById('portfolio');
+
+const portfolio = [
+  {
+    cardClass: 'tile card-1',
+    cardTitle: 'Tonic',
+    desktopImage: 'img/1.png',
+    mobileImage: 'img/1m.png',
+    cardDetails: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    portfolioStack: ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'],
+    btnInfo: 'See More',
+    popUpinfo: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    popupSeebtnInfo: 'See Live',
+    popupIcon: '',
+    popupDetails: 'See Source',
+    company: 'CANOPY',
```

Comment:
- [ ] See live and see source should be link. Kindly change it to all projects. I should click on this button and open projects live and source pages. You can put the portfolio projects link for now

Reply:
- Kindly Remove this, it is not part of the requirement, there is no link for those projects in the portfolio.  The Sees live and see source should not link anywhere. Cause those projects were not developed by the student. :smiley:  :smiley:  :smiley:

Reply:
@MaganaAsiati It is kinda [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=For%20each%20project%20you%20need%20to%20store%20the%20following%20pieces%20of%20data%20in%20a%20JavaScript%20object%2C%20at%20a%20minimum%3A%20name%2C%20description%2C%20featured%20image%2C%20technologies%2C%20link%20to%20live%20version%2C%20link%20to%20source.).
Of course these projects not developed by student that's why he can add link of the this project and in the future he can change with real projects. Also it is more makesense adding link in the button.